lms/djangoapps/certificates/api.py

```diff
@@ -172,6 +173,14 @@ def generate_user_certificates(student, course_key, course=None, insecure=False,
         forced_grade - a string indicating to replace grade parameter. if present grading
                        will be skipped.
     """
+    users_queryset = list_with_level(course, u'beta')
+    username_list = [user.username for user in users_queryset]
+
+    if student.username in username_list:
```

Comment:
Also, do we need this check in `regenerate_course_certificate()` as well?

Reply:
I think you are mentioning regeneration through support tool.If yes, then there is no need to add this check there because regenerate request is posted to 'certificates/generate' endpoint that ultimately call generate_user_certificate function to generate cert.
If I am missing any other flow, please provide git-hub code-point.